wiki/Beatmap/Pattern/common/double/en.md

```diff
@@ -0,0 +1,20 @@
+---
+stub: true
+tags:
+  - doubles
+---
+
+# Doubles
+
+A **Double** pattern is made when two [hit objects](/wiki/Gameplay/Hit_object) occur after each other in a 1/2 or faster rhythm spacing on the same position.
+
+![](img/double.jpg "Visual representation of a double in the osu! gamemode.")
+
+In [osu!](/wiki/Game_mode/osu!), doubles are when two [hit circles](/wiki/Gameplay/Hit_object/Hit_circle) are [stacked](/wiki/Beatmapping/Mapping_techniques/Stack) on top of each other.
+
+![](img/maniadouble.png "Visual representation of a double in the osu!mania gamemode.")
+
+In the [osu!mania](/wiki/Game_mode/osu!mania), Doubles are known as [jump chords](/wiki/Beatmap/Pattern/osu!mania/Chord)
+
+![](img/taikodouble.png "Visual representation of a double in the osu!taiko gamemode.")
```

Comment:
```suggestion
![](img/taikodouble.png "Visual representation of a double in the osu!taiko gamemode.")

```
Add a line break between two images.

Reply:
Will start working on fixes once I'm back home, thanks for helping

Reply:
Since the common patters were deleted untill i get more of them, you could say this is applied?